script/apply-fragments.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-continue */
+
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
```

Comment:
I think `.yml` will become a common way for us store data in `vagov-content`, so I figured the module is justified.